### ISSUE TITLE
`{{title}}` format syntax

### ISSUE THREAD

**Issue by chhoumann** [chhoumann/quickadd#327]
See #276 for details.

To emulate support for Obsidian's internal templates plugin.

**Comment by chhoumann** [comment]
The behavior of the core Templates plugin is to insert the template into the active note. [`{{title}}` is just the title of the active file](https://help.obsidian.md/Plugins/Templates).
This makes the implementation trivial - just replace `{{title}}` by `app.workspace.getActiveFile().basename`.

So it would be easy to implement for captures to the active file.
However, what about captures to some other file? And Templates? Even worse - the `format` API?

The current implementation of QuickAdd doesn't lend itself to passing knowledge of the file being created / captured to into the format syntax rendering process.
Doing so is much more involved and would require reworking core processes of QuickAdd.
I'm not against doing this - I haven't yet thought through how such a design would look.
It would just require a good amount of time to do, and I'm not sure if it's valuable enough to warrant the time invested.

For now, I advise people to use `{{value}}` (or `{{name}}` - same thing). 
This will work across captures, templates, macros, etc.

**TL;DR:** Making this work takes a bunch of time that might not be worth it. Use `{{value}}` instead for now.
